src/utils/queue_builder.rs

```diff
@@ -82,7 +88,7 @@ pub fn queue_builder(
         // If a panic occurs while building a crate, lock the queue until an admin has a chance to look at it.
         let res = catch_unwind(AssertUnwindSafe(|| {
             match build_queue.build_next_queue_package(&mut builder) {
-                Err(e) => error!("Failed to build crate from queue: {}", e),
+                Err(e) => warn!("Failed to build crate from queue: {}", e),
                 Ok(crate_built) => {
```

Comment:
from what I see this can happen when building the package gives us an `Err`. 
It seems like a normal build failure would result in just an `Ok(false)` from `build_package`. 

So that probably means we should go back to `error!` (or more exactly, `report_error`) here :) 

> Should we also be incrementing the queue attempts here?

Short answer: I don't know since I don't know this part of the codebase yet :)

Reply:
Hmm, ok - let's wait until we have sentry set up and we can decide whether to increment the attempts based on the errors we see in practice :)